src/Storages/ObjectStorage/StorageObjectStorageCluster.cpp

```diff
@@ -42,7 +44,8 @@ String StorageObjectStorageCluster::getPathSample(StorageInMemoryMetadata metada
         context,
         {}, // predicate
         {},
-        metadata.getColumns().getAll(), // virtual_columns
+        {}, // virtual_columns
+        {}, /* hive_columns */
```

Comment:
Let's be consistent with existing code
```suggestion
        {}, // virtual_columns
        {}, // hive_columns
```